app/controllers/admin/orders_controller.rb

```diff
@@ -2,11 +2,9 @@ class Admin::OrdersController < Admin::AdminController
   load_and_authorize_resource
 
   def index
-    @order = Order.search_by_status(params[:status]).newest
+    @q = Order.joins(:user).ransack params[:q].try(:merge, m: "or")
```

Comment:
ơ sao lại phải merged thêm or như này em

Reply:
em định dùng or để thay thế tìm kiếm and default của ransack nhưng khi thêm tìm kiếm theo ngày thì logic nó k đúng nữa nên em bỏ rồi